config.ts

```diff
@@ -34,11 +33,14 @@ const base = (...args) => path.resolve(...[envConfig.path_base, ...args])
 
 const paths = {
   base,
-  src: base.bind(null, envConfig.dir_src),
-  dist: base.bind(null, envConfig.dir_dist),
   dll: base.bind(null, envConfig.dir_dll),
   docsDist: base.bind(null, envConfig.dir_docs_dist),
   docsSrc: base.bind(null, envConfig.dir_docs_src),
+  packageDist: (packageName: string, ...paths: string[]) =>
```

Comment:
would expect `dist` artefacts being placed somewhere not in the `src` dir - essentially, it might be the same hierarchy of directories (by project name) provided for the `dist`

Reply:
Actually, I'm not sure that it's possible at all. Lerna will call NPM release process that requires to place file in the package folder. Did I miss something?

Reply:
no, you are not missing anything - while it is possible, it would require us to hoist paths specified in `package.json` (`files: '../dist/react'`), and then we have to select between the following:
- current approach, where all the packages are isolated from each other (although, yes, may be populated with dist artefacts)
- alternative one, where the isolation will be, essentially, not that strict, as hoisted (`../dist/react/`) file paths will be provided in the `package.json`

Thus, yes, would totally support taken approach - and if there will be true concerns around `dist` location, we might rethink. Thank you!